indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/SinglePhaseSubTask.java

```diff
@@ -262,6 +263,8 @@ public TaskStatus runTask(final TaskToolbox toolbox)
           inputSource,
           toolbox.getIndexingTmpDir()
       );
+
+      Thread.sleep(60000);
```

Comment:
what is this for?

Reply:
Ah, that was left in from testing, removed this